app.js

```diff
@@ -0,0 +1,12 @@
+const hamMenu = document.getElementById('navmenu');
+
+/* eslint-disable */
+function hamMenuShow() {
+  hamMenu.classList.remove('hidden');
+}
+
+/* eslint-disable */
+function hamMenuHide() {
+  hamMenu.classList.add('hidden');
+}
+
```

Comment:
@Reem-lab I've to linter out those functions because, otherwise, they would give the "no-unused-vars" linter error message. I've searched for the solution online, and then this is the solution I found.